src/index.html

```diff
@@ -1,13 +1,45 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__img"
+        src="./images/imac.jpeg"
+        alt=" Macbook logo"
+      >
+      <div class="card__content">
+        <span>APPLE A1419 iMac 27" Retina</span>
+        <span>5K Monoblock (MNED2UA/A)</span>
+      </div>
+      <p class="card__article">Product code: 195434</p>
+      <div class="stars">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <span class="card_review">Reviews: 5</span>
+      </div>
+      <div class="card__price">
+        <p class="price_item">Price:</p>
+        <p class="price_value">$2,199</p>
```

Comment:
CSS class name should follow BEM naming convention. Consider renaming 'price_value' to 'card__price-value'.